### PR TITLE
service: align naming

### PR DESCRIPTION
**Pull request by kopiczko**

### CODE REVIEW

File: service/controller/flannel.go [hunk 1]
```diff
@@ -14,7 +14,9 @@ import (
 	"github.com/giantswarm/flannel-operator/service/controller/v3"
 )
 
-type FrameworkConfig struct {
+const flannelControllerSuffix = ""
```
Comment (kopiczko) [inline service/controller/flannel.go]: Should be `"-flannel"` eventually.

File: service/controller/flannel.go [hunk 2]
```diff
@@ -27,7 +29,11 @@ type FrameworkConfig struct {
 	ProjectName  string
 }
 
-func NewFramework(config FrameworkConfig) (*controller.Controller, error) {
+type Flannel struct {
+	*controller.Controller
+}
```
Comment (kopiczko) [inline service/controller/flannel.go]: I just named it flannel.

Comment (xh3b4sd) [reply service/controller/flannel.go]: The concept being managed might be `network`.

File: service/controller/flannel.go (hunk 1 above)
Comment (xh3b4sd) [reply service/controller/flannel.go]: IMO we should go with the plain operator name if there is only one controller.

**Review by xh3b4sd — APPROVED** [review]

File: service/controller/flannel.go (hunk 2 above)
Comment (kopiczko) [reply service/controller/flannel.go]: Renamed to `network`

**Pull request merged**